agent/config/config_test.go

```diff
@@ -155,7 +155,16 @@ func TestConfigBoolean(t *testing.T) {
 }
 
 func TestConfigDefault(t *testing.T) {
-	cfg := DefaultConfig()
+	os.Unsetenv("ECS_DISABLE_METRICS")
```

Comment:
As it is, they're magic strings. If the intent of the test is to look at defaults, I think clearing the env is just as obvious as unsetting each (and easier to maintain in the future).